Reason: Correct lib and ldflags ordering, fixes --as-needed.
Upstream: No.
--- bbdate-0.2.4/configure.in
+++ bbdate-0.2.4/configure.in
@@ -47,12 +47,13 @@
 
 CFLAGS="$CFLAGS $X_CFLAGS"
 CXXFLAGS="$CXXFLAGS $X_CFLAGS"
-LDFLAGS="$LDFLAGS $X_LIBS $X_PRE_LIBS"
+LDFLAGS="$LDFLAGS"
+LIBS="$X_LIBS $X_PRE_LIBS"
 dnl Checks for X libraries.
-AC_CHECK_LIB(X11, XOpenDisplay, LDFLAGS="$LDFLAGS -lX11",
+AC_CHECK_LIB(X11, XOpenDisplay, LIBS="$LIBS -lX11",
              AC_MSG_ERROR(XOpenDisplay not found in -lX11))
 
-LDFLAGS="$LDFLAGS $X_EXTRA_LIBS"
+LIBS="$LIBS $X_EXTRA_LIBS"
 
 dnl Checks for header files.
 AC_HEADER_STDC
